git-work.sh

```diff
@@ -46,12 +46,19 @@ case "$1" in
   # commit current work
   commit)
     [ -z "$2" ] && ( usage && exit 1 )
-    branch=$(git symbolic-ref --short HEAD)
-    if [[ $branch =~ ^-?[0-9]+$ ]]; then
-      id="#$branch: "
-    fi
     git add .
-    git commit -am "$id$2"
+    # checks is has an id
+    if [[ "$2" =~ ^\#-?[0-9]+ ]]; then
```

Comment:
@mdbs99 The first `if` checks if issue id already exist in the message; if not exist it will be put it in the message only if the branch name is a number and it's the role of the second `if`. Maybe the code could be more optimized but I think better to do it in the future.